docs/references/kc.md

```
@@ -0,0 +1,62 @@
+- **Math source**: Chester Keltner's volatility-envelope construction,
+  modernised by Linda Raschke to use ATR rather than the original
+  high–low average.
+
+## Math summary
+With pandas-ta's default `mamode="ema"` and `tr=True` (True Range, not
+the simple high-low difference):
+
+```
+Basis_t = EMA_length(close)
+ATR_t   = RMA(true_range, length)        (Wilder smoothing of TR)
+Upper_t = Basis_t + scalar × ATR_t
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)